build/common.sh

```diff
@@ -38,23 +38,28 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
-readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat "${KUBE_ROOT}/build/build-image/cross/VERSION")"
+KUBE_BUILD_IMAGE_CROSS_TAG="$(cat "${KUBE_ROOT}/build/build-image/cross/VERSION")"
```

Comment:
This file has a LOT of `readonly FOO=...`.

We only need to alter those when they do complex assignments that may have a return value.

Lines like `readonly KUBE_BUILD_IMAGE_REPO=kube-build` we do not need to and it's not clearly worth the diff to change all of these, the linter should catch when it needs to be done (though, it clearly got smarter about this between v0.7.1 and v0.8.0).